pom.xml

```diff
@@ -578,5 +579,39 @@ THE SOFTWARE.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>surefire-with-fileleak-detector</id>
+      <properties>
+        <maven-surefire-plugin.agent>-javaagent:${project.build.directory}\agents\file-leak-detector.jar=http=20000</maven-surefire-plugin.agent>
```

Comment:
do you not want to use `/` rather than `\` otherwise this will blow up on unixy type systems